client/src/ledger_app_clients/ethereum/eip712/InputData.py

```diff
@@ -165,10 +165,7 @@ def encode_bool(value: str, typesize: int) -> bytes:
 
 
 def encode_string(value: str, typesize: int) -> bytes:
-    data = bytearray()
-    for char in value:
-        data.append(ord(char))
-    return data
+    return value.encode()
```

Comment:
This function seems useless now. it may be removed ?

Reply:
These encoding functions are called in a generic manner so the .encode() has to be wrapped like this. :+1: